docs/DeveloperGuide.md

```diff
@@ -562,19 +582,135 @@ The following activity diagram summarizes what happens when a user executes the
 <div style="page-break-after: always;"></div>
 
 #### Design consideration
-##### Aspect: Definition of number of high risk location of infection
-When displaying the list of high risk location of infection, the top few locations that has been most visited by any 
-infected people are wanted. 
+##### Aspect: User input for number of high risk locations
+The parameter for list high risk locations command is optional. Users can choose to or choose not to input the number of
+high risk locations for this command. 
+
+**Rationale**
+Instead of always using the pre-defined rule within VirusTracker, this implementation allows users to customize the
+number of high risk locations displayed in VirusTracker. If the pre-defined rule is in used, users might not be able to 
+view more infected locations beyond the displayed high risk locations that are selected automatically by VirusTracker.
+
+##### Aspect: Determining number of high risk locations for infection when user does not specify the number
+In the case when user does not specify the number of high risk locations in the command, the system itself will
```

Comment:
Small grammatical error. `In the case when the user...`

Reply:
Fixed.